.circleci/config.yml

```diff
@@ -859,7 +865,8 @@ jobs:
       - deploy-jar:
           jar_base_name: DataDonationPlatform
           deploy_config_file_dir: ../output-config
-      - update-dispatch-rules
+      - update-dispatch-rules:
+          use_current_branch_name: true
```

Comment:
The `deploy-stored-jar-job` is used for test/staging/production deploys, so we use new parameter here.